source/opt/vector_dce.cpp

```diff
@@ -377,6 +384,14 @@ bool VectorDCE::RewriteInsertInstruction(
   return false;
 }
 
+void VectorDCE::MarkDebugValueUsesAsDead(Instruction* composite) {
+  context()->get_def_use_mgr()->ForEachUser(
+      composite, [this](Instruction* use) {
+        if (use->GetOpenCL100DebugOpcode() == OpenCLDebugInfo100DebugValue)
```

Comment:
Change to `if (use->IsOpenCL100DebugInstr())`?

Reply:
No. I want to check if this is exactly `DebugValue` not other debug instructions.

Reply:
Ah, yes.